isd-backend/src/main/java/com/example/isdbackend/service/UserService.java

```diff
@@ -0,0 +1,66 @@
+package com.example.isdbackend.service;
+
+import com.example.isdbackend.model.Menu;
+import com.example.isdbackend.model.Order;
+import com.example.isdbackend.model.Provider;
+import com.example.isdbackend.model.User;
+import com.example.isdbackend.repository.UserRepository;
+import org.aspectj.apache.bcel.classfile.Module;
+import org.springframework.stereotype.Service;
+
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.time.DayOfWeek;
+import java.util.*;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
+@Service
+public class UserService {
+    private UserRepository userRepository;
+    private ProviderService providerService;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public User getUser(Long id){
+        return userRepository.findById(id).orElseThrow();
+    }
+
+    public void notification(Long id, Boolean active){
```

Comment:
We should do this in save method,  call setNotificationSettings(notificationSettings) on the user object, we set the default values for user notification settings for ex: active field as true others as null.